app/models/wizards/signup/sektion_operation.rb

```diff
@@ -77,25 +67,20 @@ def generate_invoice
       invoice = ExternalInvoice::SacMembership.create!(
         person: person,
         state: :draft,
-        year: invoice_date.year,
-        issued_at: invoice_date,
-        sent_at: invoice_date,
+        year: today.year,
+        issued_at: today,
+        sent_at: today,
         link: role.layer_group
       )
-      Invoices::Abacus::CreateInvoiceJob.new(invoice, invoice_date, new_entry: true).enqueue!
+      Invoices::Abacus::CreateInvoiceJob.new(invoice, today, new_entry: true).enqueue!
     end
 
     def neuanmeldung?
       group.is_a?(Group::SektionsNeuanmeldungenSektion) ||
         group.is_a?(Group::SektionsNeuanmeldungenNv)
     end
 
-    def invoice_date
```

Comment:
This calculation isn't needed anymore, since all invoices, should get sent out right after the registration is completed